app/controllers/users_controller.rb

```diff
@@ -41,11 +41,11 @@ def get_all_users
   end
 
   def get_users
```

Comment:
@itc02 
Es methodnen `get_users` and `user_todos_number` hunch chukechi Privata.
Lav kini orinak obshi `private` tanq.